index.html

```diff
@@ -136,6 +136,37 @@ <h2>A project for first-time contributions</h2>
         <!-- make sure there is one line of space above and below your card -->
         <!-- ========== ⬇⬇ Paste YOUR CARD directly BELOW this line ⬇⬇ ========== -->
 
+        <!-- ________ ffGambit card START ________  -->
+        <div class="card">
+          <p class="name">ffGambit</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="www.twitter.com" target="_blank"></a>
```

Comment:
You haven't added your profile link to Twitter.

Reply:
I'm fine without. Do I need to go create an account for this?

Reply:
No need to create a Twitter account, Thank you for your contribution.🚀Best of luck in open source. 🎉